src/test/scala/sigmastate/serialization/AndSerializerSpecification.scala

```diff
@@ -1,19 +1,19 @@
 package sigmastate.serialization
 
-import sigmastate.{AND, SBoolean}
+import sigmastate.AND
 import sigmastate.Values.BooleanConstant
-import OpCodes._
+import sigmastate.serialization.OpCodes._
 
 class AndSerializerSpecification extends TableSerializationSpecification {
 
   def boolConst(b: Boolean): BooleanConstant = BooleanConstant.fromBoolean(b)
 
   override def objects = Table(
     ("object", "bytes"),
-    (AND(boolConst(true), boolConst(true)),  Array[Byte](AndCode, ConcreteCollectionCode, 0, 2, SBoolean.typeCode, TrueCode, TrueCode)),
-    (AND(boolConst(true), boolConst(false)), Array[Byte](AndCode, ConcreteCollectionCode, 0, 2, SBoolean.typeCode, TrueCode, FalseCode)),
-    (AND(boolConst(false),boolConst(true)),  Array[Byte](AndCode, ConcreteCollectionCode, 0, 2, SBoolean.typeCode, FalseCode, TrueCode)),
-    (AND(boolConst(false),boolConst(false)), Array[Byte](AndCode, ConcreteCollectionCode, 0, 2, SBoolean.typeCode, FalseCode, FalseCode))
+    (AND(boolConst(true), boolConst(true)),  Array[Byte](AndCode, 0, 2, TrueCode, TrueCode)),
+    (AND(boolConst(true), boolConst(false)), Array[Byte](AndCode, 0, 2, TrueCode, FalseCode)),
+    (AND(boolConst(false),boolConst(true)),  Array[Byte](AndCode, 0, 2, FalseCode, TrueCode)),
+    (AND(boolConst(false),boolConst(false)), Array[Byte](AndCode, 0, 2, FalseCode, FalseCode))
   )
```

Comment:
@greenhat I expected these cases serialize into 4 bytes as result of using VLQ encoding of collection length.

Reply:
@ergomorphic Do you want to introduce `put/getUShort`? `putUInt` will do the job but will look kinda sloppy given we limit collection size to `Short.MaxValue`.

Reply:
yes, put/getUShort is required, because we have limit of 65536 items in a collection.
Of case in the API it we should use Int `def getUShort: Int`

Reply:
Agreed. That also means that we should have `def getUInt:  Long` I suppose.

Reply:
yeah, that is right